src/main/java/se/sundsvall/supportmanagement/service/scheduler/supensions/SuspensionWorker.java

```diff
@@ -32,7 +32,13 @@ public void processExpiredSuspensions() {
 		errandsRepository
 			.findAllBySuspendedToBefore(now())
 			.forEach(entity -> {
-				if (!notificationService.doesNotificationWithSpecificDescriptionExistForOwnerAndErrand(entity.getMunicipalityId(), entity.getNamespace(), entity.getAssignedUserId(), entity, NOTIFICATION_MESSAGE)) {
+
+				if (!entity.getStatus().equals("SUSPENDED")) {
+					return;
+				}
```

Comment:
Status should probably never be null, but this would throw a NPE if it was. Switch it around to avoid NPE's 

`!"SUSPENDED".equals(entity.getStatus)`